### PR TITLE
fix cli kwargs again??

### PR DESCRIPTION
**Pull request by hackrush01**

### CODE REVIEW

File: lbrynet/daemon/DaemonCLI.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def main():
     # TODO: check if port is bound. Error if its not
 
     try:
-        result = api.call(method, kwargs)
+        result = api.call(method, **kwargs)
```
Comment (jackrobison) [inline lbrynet/daemon/DaemonCLI.py]: `call` doesn't take variable keyword args: https://github.com/lbryio/lbry/blob/4e7bdafdcfb80419234a5db0dcce12134f015a7b/lbrynet/daemon/auth/client.py#L48

**Comment by jackrobison** [comment]
Can you post the error? These commands run fine for me with and without kwargs currently, and with this change would raise `TypeError: call() got an unexpected keyword argument` with the keyword arg

**Comment by hackrush01** [comment]
@jackrobison 
```
lbrynet-cli status
ERROR: sequence item 0: expected string, dict found
```

```
2018-04-18 19:54:09,408 ERROR    lbrynet.daemon.auth.server:227: sequence item 0: expected string, dict found
2018-04-18 19:54:09,408 WARNING  lbrynet.daemon.auth.server:211: error processing api request: sequence item 0: expected string, dict found
traceback:
```

**Comment by jackrobison** [comment]
fixed by https://github.com/lbryio/lbry/commit/450bed6350f3d4aab3a49432a59f45f62bad69ec

**Pull request closed without merging**